### ISSUE TITLE
the train_mean_loss

### ISSUE THREAD

**Issue by tht106** [Levigty/AimCLR#14]
Hello, thank you for presenting your excellent work! 

While pretraining aimclr,  how do you measure the progress with respect to the loss function train_mean_loss, what is a good value demonstrating that we can stop the aimclr pretraining to yield a good encoder?

Thanks
